core/spec/models/concerns/spree/has_image_alt_text_spec.rb

```
@@ -0,0 +1,82 @@
+require 'spec_helper'
+
+RSpec.describe Spree::HasImageAltText, type: :concern do
+  # Create a dummy class to test the concern
+  let(:dummy_class) do
+    Class.new do
+      include Spree::HasImageAltText
+      
```

RuboCop (1.76.1)

[convention] Trailing whitespace detected.

(Layout/TrailingWhitespace)